src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
+    val LOG = Logger.getLogger(this.javaClass.name)
+    private const val TARGET_PATH = "/tmp/pythonparser"
+
+    fun putParserToTmp() {
+        //copy parser file into /tmp directory, required by Gumtree
+        try {
+            LOG.info("Putting parser into $TARGET_PATH")
+
+            val pythonparserFile = File(javaClass.getResource("./parser/pythonparser.py").path)
+            val targetFile = File(TARGET_PATH)
+
+            pythonparserFile.copyTo(targetFile)
+            makeFileExecutable()
```

Comment:
Not sure that we need to call "makeFileExecutable" inside "putParserToTmp" because it's a separated action that doesn't relate to putting files to tmp.